src/Hazelcast.Net/Protocol/Codecs/ClientAuthenticationCodec.cs

```diff
@@ -198,10 +198,21 @@ public sealed class ResponseParameters
             /// Returns true if server supports clients with failover feature.
             ///</summary>
             public bool FailoverSupported { get; set; }
+
```

Comment:
leaking some Alto changes that should not be part of this PR

Reply:
Let me also add this for the record. The changes will be part of 5.3 which covers alto and SQL partition aware feature, and codecs cover both cases.

Reply:
Regenerated the files with latest protocol at https://github.com/hazelcast/hazelcast-csharp-client/pull/798/commits/a91d960491af90459c71f0821c8ccfdd5e85d119